packages/backend/server/routes/stats.get.ts

```diff
@@ -0,0 +1,47 @@
+export default eventHandler(async (event) => {
+  try {
+    const binding = useBinding(event);
+
+    let cursor: string | undefined;
+    let objectCount = 0;
+    const orgs = new Set<string>();
+    const repos = new Set<string>();
+    const commits = new Set<string>();
+    const packages = new Set<string>();
+
+    const prefix = `${usePackagesBucket.base}:`;
+
+    do {
+      const { objects, truncated, cursor: nextCursor } = await binding.list({ prefix, cursor });
+      objectCount += objects.length;
+
+      for (const { key } of objects) {
+        const trimmedKey = key.slice(prefix.length);
+        const parts = trimmedKey.split(":");

```

Comment:
what happened to https://github.com/stackblitz-labs/pkg.pr.new/pull/288/files#r1855429078 ?